### PR TITLE
Add docs for D2E relay.

### PR DESCRIPTION
**Pull request by wi1dcard**
@HackFisher @wuminzhe Please take a look, see if there's anything I should add or remove.

### CODE REVIEW

**Review by wuminzhe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -89,6 +89,37 @@ real = "0x0000000000000000000000000000000000000000000000000000000000000000"
...
+2. If you're not using a proxy account or you have the permission to sign extrinsics using the "real" account, switch to the "real" account and submit `ethereumRelayAuthorities.requestAuthority(stake_amount, signer)`:
+   - `stake_amount` is the amount of RINGs to stake.
+   - `signer` is the public key of your **Ethereum** account.
+    Otherwise, you must switch to your proxy account and submit `proxy.proxy(real_account, ethereumRelayAuthorities.requestAuthority(stake_amount, signer))`. The `stake_amount` will be deducted from your "real" account.
+3. Notify council members to submit `ethereumRelayAuthorities.addAuthority(your_account)`.
+
+> How it works: Authority is a role of relayer. Authorities sign Darwinia-to-Ethereum messages. Once the number of signatures reaches the threshold and the message is delivered and verified on the Ethereum network, the cross-chain is finished. Updating the authority set involves 2 times of cross-chain: 1) relay the new authority set from Darwinia to Ethereum; 2) relay from Ethereum to Darwinia to send rewards to `darwinia_to_ethereum.beneficiary` (see below).
```
Comment (wuminzhe) [inline README.md:110]: "Authority is a role of relayer"
I feel not good, but I don't know how to describe it better.

Comment (HackFisher) [reply README.md:110]: Authority is not relayer, authority is the validators/nodes in source chain consensus system to resolve Byzantine Problem, and finalize the blocks.

Grandpa authorities are BFT alike authorities, our authority concept come from the similar meaning, is to be used as replacement for grandpa authorites.

@wuminzhe @wi1dcard

Comment (HackFisher) [reply README.md:110]: Relayer(aka. Bridger) are offchain worker clients who help relay the headers and messages between source chain and target chain, they are working in between two chains and need to access rpcs from two chains.

Our `bridger` repo is an implementation of relayer client.

Comment (wi1dcard) [reply README.md]: @HackFisher Fixed, please review

**Review by HackFisher — APPROVED** [review]

**Pull request merged**